docs/source/contribution.rst

```diff
@@ -171,6 +171,9 @@ In order to run unit tests at the repository root, you first have to build Cytho
 
   See `ccache <https://ccache.samba.org/>`_ for details.
 
+  If you want to use ccache for nvcc, please install ccache v3.3 and later.
+  And, please set environment variable `NVCC=/path/to/nvcc_with_ccache`.
```

Comment:
How about:
..., please install ccache v3.3 **or** later and set environment variable ...